scheduler/src/cook/mesos/util.clj

```diff
@@ -150,15 +150,21 @@
 (def ^:const default-job-priority 50)
 
 (defn same-user-task-comparator
-  "Comparator to order same user's tasks"
-  [task1 task2]
-  (letfn [(task->feature-vector [task]
+  "Comparator to order same user's tasks
+
+   If consider-backfilled-jobs? is true, then we treat jobs which were backfilled as being the lowest priority"
+  [consider-backfilled-jobs?]
```

Comment:
Yeah, could make this 2 separate functions.  I believe the one that cares about backfill could just use the result of the 2nd function as a "tiebreaker" comparator between jobs that share a backfill value